src/Me/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,72 @@
+import React, { useContext } from 'react';
+import { formFieldContext } from '../FormField/formContext';
+import styled from 'styled-components';
+
+const CheckboxContainer = styled.div`
+  width: fit-content;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  cursor: pointer;
+  position: relative;
+  vertical-align: middle;
+`;
+
+const Icon = styled.svg`
+  fill: none;
+  stroke: #179a6f;
+  stroke-width: 2px;
+  visibility: hidden;
+  transition: all 0.1s ease-in-out;
+`;
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+  opacity: 0;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+`;
+
+const StyledCheckbox = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  box-sizing: border-box;
+  padding-right: 2px;
+  border: 2px solid #69d5b1;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
+  background-color: #fff;
+  cursor: pointer;
+
+  ${Icon} {
+    visibility: ${props => (props.checked ? 'visible' : 'hidden')};
+  }
+`;
+
+const Label = styled.label`
+  margin-left: 0.9rem;
+  color: #4f4f4f;
+  font-size: 14px;
+`;
+
+export const Checkbox = ({ LabelComponent, ...props }) => {
+  const id = useContext(formFieldContext);
+  return (
+    <CheckboxContainer>
+      <HiddenCheckbox id={id} checked={props.checked} {...props} />
```

Comment:
```suggestion
      <HiddenCheckbox id={id} defaultChecked ={props.checked} {...props} />
```
To avoid

> react-dom.development.js:530 Warning: A component is changing an uncontrolled input of type checkbox to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). 

https://stackoverflow.com/a/32174344/863110